src/plugins/unbounce-event-mixin.js

```diff
@@ -17,30 +20,53 @@ export default {
 		}
 
 		const utmEmailMediumCookie = this.cookieStore.get('kvutm_medium_email');
+		// default to true so that we don't fire the unbounce event if we can't read the query
+		let hasEverLoggedIn;
+
+		/** get has ever logged in query */
+		try {
+			const data = this.apollo.readQuery({
+				query: hasEverLoggedInQuery
+			});
+			// Setup unbounce event trigger which is restricted to non-logged-in and unknown users
+			hasEverLoggedIn = data?.hasEverLoggedIn ?? true;
+			// Check for hasEverLoggedIn
+			if (!hasEverLoggedIn) {
+				// push data object if eligible + assigned
+				if (typeof window !== 'undefined') {
+					window.dataLayer.push({ event: 'activateUnbounceEvent' });
+				}
```

Comment:
I think we ened to remove this block and let the code below handle it when `unbounceRoute` is active.

Also, how are we going to gate this so it's not firing all over the site and in checkout and on settings pages? I think `activateUnbounceDonation` should also be gated by `unbounceRoute`

Reply:
All of the restrictions we put on the original event seem like they should apply here too. If someone is coming to the site from one of our emails they likely have a goal already and it may already be donating so we prob shouldn't get in the way.

Reply:
Will remove block, I did not mean to have it twice. 

I think the idea was to gate it on the unbounce side. But I will add the meta property, for redundancy

Reply:
@mcstover I circled back with the team, and the desired behavior varies from the behavior of the email capture modal. 

To be clear, this modal will have the unbounce rule of "When a visitor tries to exit the page." (Im not sure what that means technically, I assume its a window unload event)

* The modal should only load for hasLoggedInUsers
* The modal should fire on a couple of pages, specifically the thank you page (which currently does not have the `unbounce: true` meta property)

Reply:
Ok can we at least target a small set of specific pages in the Unbounce setup? This shouldn't be firing anywhere within portfolio or settings or teams or any of the Ui borrower onboarding process or on checkout and def NOT on the `/cc/` routes for managed lending...